.github/workflows/update-latest.yml

```diff
@@ -81,7 +81,8 @@ jobs:
       - name: Generate the latest spec
         run: |
           source localstack-ext/.venv/bin/activate
-          python bin/update_latest.py
+          pip install click
```

Comment:
Q: Is this necessary? Click should already be a transitive dependency through localstack, right?
In any case, would it make sense to add this to a `requirements.txt` file next to the script to make this an explicit dependency of the script?